### PR TITLE
Add test for SECURITY-3135

### PR DESCRIPTION
**Pull request by Kevin-CB**
Forgotten follow-up for [SECURITY-3135](https://www.jenkins.io/security/advisory/2023-06-14/#SECURITY-3135), a regression test.

### Desired reviewers

@daniel-beck 
or
@yaroslavafenkin 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8427"></a>

### CODE REVIEW

**Review by yaroslavafenkin — APPROVED** [review]

File: test/src/test/java/jenkins/security/Security3135Test.java
Comment (daniel-beck) [inline test/src/test/java/jenkins/security/Security3135Test.java:30]: This test looks unconvincing to me, in particular it looks like it would continue to pass with UI rework resulting in the `click`ed element being one without associated behaviors. Some options:

* Assert the HTTP request/response as a result of the interaction.
* Have a corresponding "good case" case with the same.
* Assert something about this element, so the test starts failing during UI rework.

Additionally, commenting out https://github.com/jenkinsci/jenkins/blob/8041e14c42084930c7b8ade2b7ffb2b2b85576b4/core/src/main/java/jenkins/model/ParameterizedJobMixIn.java#L211 should fail the test (AFAICT), but does not. Interactive testing indicates that aggregated URL contains an unparseable delay string due to concatenation, so results in HTTP 500 (`java.lang.IllegalArgumentException: Invalid time duration value: 0sec/contextMenu?delay=0`).

**Review by daniel-beck — CHANGES_REQUESTED** [review]

**Review by daniel-beck — APPROVED** [review]

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
